waiter/src/waiter/request_log.clj

```diff
@@ -0,0 +1,52 @@
+;;
+;;       Copyright (c) 2018 Two Sigma Investments, LP.
+;;       All Rights Reserved
+;;
+;;       THIS IS UNPUBLISHED PROPRIETARY SOURCE CODE OF
+;;       Two Sigma Investments, LP.
+;;
+;;       The copyright notice above does not evidence any
+;;       actual or intended publication of such source code.
+;;
+(ns waiter.request-log
+  (:require [clj-time.coerce :as coerce]
+            [clojure.string :as str]
+            [waiter.utils :as utils]))
+
+(def ^:dynamic *request-log-file* "log/request.log")
+
+(defn log
+  "Log a request context."
+  [context]
+  (spit *request-log-file* (str context \newline) :append true))
```

Comment:
Should we use the logging library to do this?

Reply:
Yes.